dapps/settings/src/components/browser/browser.vue

```diff
@@ -0,0 +1,97 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <div class="evan-content-container">
+    <div class="white-box mt-5">
+      <h3 class="mb-5 d-block">
+        {{ `_settings.browser.language` | translate }}
```

Comment:
We should generally discuss which form of translation to prefer. Currently $t(...) is better for me (and maybe Karl too), because only that works with the i18n ally extension.

On that note: Is there any reason we added the custom translate pipe?

Reply:
I have no problem with the `$t` function, it's a old habit from angular. I will add `$t` in future, where ever i need a translation.